README.md

```diff
@@ -104,6 +104,8 @@ some extensions or allowing some deviations from the specification.
 
  * With the flag `MD_FLAG_NOINDENTEDCODEBLOCKS`, indented code blocks are
    disabled.
+   
+ * With the flag `MD_FLAG_REDDITSLASHDETECTION`, Reddit subreddit and user links such as r/test or /u/me are detected
```

Comment:
Please rename the flag for consistency with other flags. No other flag uses the word "detection". I am not reddit user, so IDK whether something like `MD_FLAG_REDDITAUTOLINKS` but it could give you an idea.